source/_includes/_adminpanel/domains-ce.rst

```diff
@@ -249,12 +252,93 @@ CERTIFICATE`. In the dialog, you can choose to use:
   notification will be shown (:bdg-success:`The certificates have been
   saved`). To complete the procedure: if you are on a Single-Node,
   restart it otherwise you need to restart the node on which the
-  **Proxy** is installed;
-
+  **Proxy** is installed.
 
 You can :red:`REMOVE` or :blue:`DOWNLOAD` the certificates
 by clicking the appropriate button above the certificates themselves.
-                  
+
+.. _le-procedure:
+
+Procedure to install a Let's Encrypt certificate
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. card:: Let's Encrypt Requirements
+
+   Before attempting to ask for a Let's Encrypt certificate, make sure
+   that:
+
+   #. **Public Service Protocol** and **Public Service Host Name** are
+      correctly set in the |adminui|'s :menuselection:`Domain -->
+      General Settings`
+
+   #. There is a Virtual Host correctly configured for the domain you
+      want the certificate
+
+   #. **A**, **AAAA**, and **NXDOMAIN** record in the domain's DNS
+      configuration
+
+   #. The domain has a valid |fqdn| that can be resolved from anywhere
+      (i.e., the domain must be publicly accessible)
+
+   #. The domain (i.e., the Node which installs the Proxy) is
+      reachable from the Internet. In case the proxy is behind a
+      firewall or other routing devices and can not be reached
+      directly, add some forwarding rule on the firewall or device.
+
+To correctly issue a Let's Encrypt certificate for your |product|
+installation, you should carry out the following steps.
+
+The starting point is to generate the certificate using the |adminui|
+button, as shown in the :ref:`previous section <ap-vhost>`. besides
+the message on the bottom right corner, you will receive in a few
+minutes an e-mail stating the success or failure of the certificate's
+generation.
+
+.. hint:: You can follow the process by checking the log file
+   :file:`/var/log/carbonio/letsencrypt/letsencrypt.log`, using the
+   :command:`tail -f` command from the CLI.
+
+In case of failure, the e-mail will report the errors encountered that
+you need to fix before attempting again. Take into account that if you
+continuously ask for a certificate without success, you can be
+temporarily be prevented to ask again.
+
+The message *Successfully received certificate* appears in the e-mail
+when the issue is successful, together with other information,
+including the expiry date, followed by a second confirmatory e-mail.
```

Comment:
...related to the previous point about the notification attribute (more testing needed)